### ISSUE TITLE
Scrum Report not working in CMMI model

### ISSUE THREAD

**Issue by bryan-vencore** [mikaelkrief/VSTS-Extension-ScrumReport#9]
We were using Scrum Report in the TFS Scrum model and then switched to the CMMI model - now Scrum Report doesn't work for us - the Scrum Report page never populates (but it's fun watching the spinner spin ...)

I uninstalled the extension and re-installed it - that didn't make any difference

Yes, I have a Scrum Report query defined in the Shared Queries area

**Comment by mikaelkrief** [comment]
Ok, I will investigate

**Comment by bryan-vencore** [comment]
Let me know if there are any logs or messages I can provide you

**Comment by mikaelkrief** [comment]
yes, could you send me browser devellepper tools log screenshoot

**Comment by bryan-vencore** [comment]
Is this what you were looking for, Mikael?

The error displayed in the log makes me think the code is trying to loop on the results of the ‘Scrum Report’ query but can’t find the query or can’t find the results

**Comment by mikaelkrief** [comment]
have you screenshoot of this log?

**Comment by bryan-vencore** [comment]
From: Post, Bryan
Sent: Monday, September 19, 2016 9:49 AM
To: 'mikaelkrief/VSTS-Extension-ScrumReport'
Subject: RE: EXTERNAL: Re: [mikaelkrief/VSTS-Extension-ScrumReport] Scrum Report not working in CMMI model (#9)

Is this what you were looking for, Mikael?

The error displayed in the log makes me think the code is trying to loop on the results of the ‘Scrum Report’ query but can’t find the query or can’t find the results
